docs/grammars.md

```diff
@@ -39,7 +39,7 @@ FALSE.2: /false/
 
 // Symbols cannot contain uppercase letters, because these are reserved
 // Moreover, any word between quotes is a symbol
-SYMBOL_NAME: /[a-z][a-z0-9_]*|"\w+"/
+SYMBOL_NAME: /[a-z][a-z0-9_-]*[a-z0-9_]*|"\w+"/
```

Comment:
this does not look right.

See e.g. here: https://regex101.com/r/NYgppM/1 ; `a-` is accepted.

Reply:
True! It was not that a simple fix ;)

What about this one? 
`^(?:"[^"]*"|(?:[a-z][\w-]*[a-z]|[a-z]))$`